app/src/components/swap/header.tsx

```diff
@@ -17,17 +17,23 @@ import BraveLogoDark from '../../assets/brave-logo-dark.svg'
 import {
   ThemeButton,
   SelectTokenOrNetworkButton,
-  NetworkListButton,
   ConnectWalletButton
 } from '../buttons'
+import { NetworkSelector } from './network-selector'
 
 // Hooks
 import { useWalletState, useWalletDispatch } from '../../state/wallet'
 
 // Styled Components
 import { Row, HorizontalSpacer } from '../shared.styles'
 
-export const Header = () => {
+interface Props {
+  getNetworkFeeFiatEstimate: (network: NetworkInfo) => string
```

Comment:
Are we planning to provide this via context? Looks like this is being used for prop-drilling

Reply:
Removed as prop!